File: Examples/DotNET/SDK/VisualBasic/Bookmark/GetAllBookmarks.vb
Imports Aspose.Cloud

Namespace Bookmark
    Class GetAllBookmarks
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample.doc"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, String.Empty)

            ' Get all Bookmarks from a Word
            Dim wordsBookmarksResponse As WordsBookmarksResponse = Common.WordsService.WordsBookmarks.ReadDocumentBookmarksCommonInfo(inputfile, Common.FOLDER)

            Console.WriteLine((Convert.ToString(vbLf) & inputfile) + " have " + wordsBookmarksResponse.Bookmarks.BookmarkList.Count + " bookmarks")
        End Sub
    End Class
End Namespace

File: Examples/DotNET/SDK/CSharp/Bookmark/GetAllBookmarks.cs
using System;
using Aspose.Cloud;

namespace CSharp.Bookmark
{
    class GetAllBookmarks
    {
        public static void Run()
        {
            string inputfile = "doc-sample.doc";
     
            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, string.Empty);

            // Get all Bookmarks from a Word
            WordsBookmarksResponse wordsBookmarksResponse = Common.WordsService.WordsBookmarks.ReadDocumentBookmarksCommonInfo(inputfile, Common.FOLDER);

            Console.WriteLine("\n" + inputfile + " have " + wordsBookmarksResponse.Bookmarks.BookmarkList.Count + " bookmarks"); 
        }
    }
}
